src/Symfony/Component/WebLink/Link.php

```diff
@@ -180,6 +180,11 @@ public function getAttributes(): array
         return $this->attributes;
     }
 
+    public function getAttribute(string $attribute): string|\Stringable|int|float|bool|array|null
+    {
+        return $this->attributes[$attribute] ?? null;
```

Comment:
`false` could be returned instead of `null`. It has the semantics of "attribute not set".
```suggestion
        return $this->attributes[$attribute] ?? false;
```